<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TokenWidget</class>
 <widget class="QWidget" name="TokenWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QPushButton" name="btnActionN">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>N</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="optTopSide">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Top Side</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="optBottomSide">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Bottom Side</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAccept">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>118</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Accept</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnActionNE">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>117</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>N</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnActionE">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>N</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnActionS">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>N</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnActionW">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>N</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnActionSE">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>182</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>N</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnActionSW">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>N</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnActionNW">
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>119</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>N</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBaseValue">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblPoints">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>231</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Points:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbTemplates">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>280</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>284</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Templates:</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
